src/example/com/example/components/datomic.clj

```diff
@@ -1,57 +1,77 @@
 (ns com.example.components.datomic
   (:require
-    [com.fulcrologic.rad.database-adapters.db-adapter :as dba]
-    [com.fulcrologic.rad.database-adapters.protocols :as dbp]
-    [com.fulcrologic.rad.database-adapters.datomic :as datomic-adapter]
-    [com.fulcrologic.guardrails.core :refer [>defn =>]]
-    [com.example.schema :refer [prior-schema latest-schema]]
-    [taoensso.timbre :as log]
-    [datomic.api :as d]
-    [datomock.core :refer [mock-conn]]
-    [mount.core :refer [defstate]]
-    [taoensso.timbre :as log]
-    [com.example.components.config :refer [config]]
-    [clojure.string :as str]))
+   [com.fulcrologic.rad.database-adapters.db-adapter :as dba]
```

Comment:
2 spaces of indent...sorry, this isn't something I want in any PR...not flip flop on indents.  In general don't commit ws changes, and fix your indentation to match the project: indentation is 2 spaces for each level of nesting.

Reply:
No worries, I see things the same way. The thing is I wanted to mimic your IntelliJ indentation setup, but I found different styles in different files (or so I thought) so I picked one. I might've missed something. I'll change it to 2 spaces 😉